Oxide.Core/RemoteConsole/RemoteConsole.cs

```diff
@@ -25,6 +27,8 @@ public void Initalize()
         {
             if (!config.Enabled) return;
 
+            if (listener != null || server != null) return;
```

Comment:
Blank line above this can be moved or even merged with previous if config check.